bundles/sirix-core/src/main/java/org/sirix/access/LocalJsonDatabaseFactory.java

```diff
@@ -0,0 +1,27 @@
+package org.sirix.access;
+
+import org.sirix.access.json.JsonResourceStore;
+import org.sirix.api.Database;
+import org.sirix.api.json.JsonResourceManager;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * A database session factory for Json databases.
+ *
+ * @author Joao Sousa
+ */
+@Singleton
+public class LocalJsonDatabaseFactory implements LocalDatabaseFactory<JsonResourceManager> {
```

Comment:
I know that this sounds dull at this point, but future refactors will definitely make this worth it!